apps/frontend/src/libs/components/balance-wheel-chart/balance-wheel-chart.tsx

```diff
@@ -71,8 +71,8 @@ const BalanceWheelChart: React.FC<Properties> = ({
 	);
 
 	const handleAnimateChart = useCallback(() => {
-		const chartInstance = chartReference.current;
 		const FIRST_ITEM_INDEX = 0;
```

Comment:
Move FIRST_ITEM_INDEX to shared

Reply:
I noticed that the constant `FIRST_ITEM_INDEX` means the same as `ZERO_INDEX`, but less used. So I think I should replace `FIRST_ITEM_INDEX` with `ZERO_INDEX`. It will affect 1 change in the file on the mobile part of the app. Is it ok if I do this?